### DISCUSSION TITLE
什么翻译软件好用啊？

### DISCUSSION THREAD

**Discussion by zRich** [lfapac-open-source-evangelist/translation#407] [Announcements]
## 👋 大家有什么好用的翻译软件推荐吗？

  现在机器翻译的水平以及很高了。根据你的使用经验推荐翻译软件，并说出它有哪些优点。尽量包含：
  * 翻译质量
  * 翻译速度
  * 使用方式（浏览器/插件/软件）
  * 是否收费，如收价格是？

**Comment by zRich** [comment]
我试了一下昨天会上 @node 说的这个“沉浸式翻译”。这是一个 Chrome 的插件。可以通过 [Immersive Translate](https://immersivetranslate.com/) 安装


支持不同的翻译服务。
![image](https://github.com/lfapac-open-source-evangelist/translation/assets/2752413/acb044b7-dcd3-489a-8c3a-6e23e959dd5f)
有收费的，也有免费的。我用免费的试着翻译了一个PDF文档，速度相当快！

左右对照，看起来很舒服。

文本翻译格式跟原文格式保持一致。
![image](https://github.com/lfapac-open-source-evangelist/translation/assets/2752413/3e97a3fa-0b45-40f6-9f86-508a0a1d8dad)

图片上的文字也可以翻译，格式上有点丑。

![image](https://github.com/lfapac-open-source-evangelist/translation/assets/2752413/28858669-9e6a-42de-8f89-a406e8632d6e)

**Comment by hwang37** [comment]
直接用google网站的translate功能，例如，https://translate.google.com/?sl=en&tl=zh-CN&op=translate
但是有些翻译出来的文字会有明显的“机器翻译”的痕迹，读起来非常拗口。

我尝试了使用AI功能进行翻译，效果好些。它会把“直译”变成”意译“，读起来效果会好些。例如：
1. Google的， https://gemini.google.com/app
2. Microsoft的，https://copilot.microsoft.com/
目前都是免费了，只需要sign in即可。

![image](https://github.com/lfapac-open-source-evangelist/translation/assets/36193324/81acc03c-f8fc-4b39-b6ce-a4acc3d43f33)

![image](https://github.com/lfapac-open-source-evangelist/translation/assets/36193324/2f08f34a-1a0c-4612-bfd7-f52fdba4d33e)
